bundles/org.openhab.binding.neohub/README.md

```diff
@@ -33,21 +33,31 @@ It signs on to the hub using the supplied connection parameters, and it polls th
 The NeoHub supports two Application Programming Interfaces "API" (an older "legacy" one, and a modern one), and this binding can use either of them to communicate with it.
 Before the binding can communicate with the hub, the following Configuration Parameters must be entered.
 
-| Configuration Parameter | Description                                                                                 |
-|-------------------------|---------------------------------------------------------------------------------------------|
-| hostName                | Host name (IP address) of the NeoHub (example 192.168.1.123)                                |
-| portNumber              | Port number of the NeoHub (Default=4242)                                                    |
-| pollingInterval         | Time (seconds) between polling requests to the NeoHub (Min=4, Max=60, Default=60)           |
-| socketTimeout           | Time (seconds) to allow for TCP socket connections to the hub to succeed (Min=4, Max=20, Default=5) |
-| preferLegacyApi         | ADVANCED: Prefer the binding to use older API calls; if these are not supported, it switches to the new calls (Default=false) |
+| Configuration Parameter    | Description                                                                                              |
+|----------------------------|----------------------------------------------------------------------------------------------------------|
+| hostName                   | Host name (IP address) of the NeoHub (example 192.168.1.123)                                             |
+| useWebSockets<sup>1)</sup> | Use secure WebSockets to connect to  the NeoHub (example `true`)                                         |
+| apiToken<sup>1)</sup>      | API Access Token for secure connection to hub. Create the token in the Heatmiser mobile App              |
+| pollingInterval            | Time (seconds) between polling requests to the NeoHub (Min=4, Max=60, Default=60)                        |
+| socketTimeout              | Time (seconds) to allow for TCP socket connections to the hub to succeed (Min=4, Max=20, Default=5)      |
+| preferLegacyApi            | ADVANCED: Prefer to use older API calls; but if not supported, it switches to new calls (Default=false)  |
+| portOverride<sup>2)</sup>  | ADVANCED: Port number for connection to the NeoHub (0 = automatic)                                       |
```

Comment:
Can we rename this back to `portNumber` for backwards compatibility? Also, I think the original name is still perfectly understandable.

Reply:
> Can we rename this back to portNumber for backwards compatibility?

No.

In existing installations, the users will have defined a portNumber config parameter (normally 4242). In the new version the binding automatically selects the port based on the useWebSockets parameter. So we have to prevent that predefined portNumber config parameter from preventing the automatic selection.

Reply:
OK, I'll have to try out how this works for managed things, if default values will actually be written? This doesn't make sense, since it would then by definition override the default value after saving, and there the value would no longer be defaulted.

Under the assumption that default value will not be written, I guess the most likely scenario for users configuring this parameter would be overriding default port, in which they probably still want the port to be overwritten after upgrading to 3.4? There's no real reason for anyone to configure the port number to be the same as the default port number for this optional parameter.

Reply:
Ok. Understood. I will try another commit.